index.bs

```diff
@@ -5892,6 +5932,16 @@ The [=type name=] of the
 {{octet}} type is "<code>Octet</code>".
 
 
+<h4 id="idl-bigint" interface>bigint</h4>
```

Comment:
This is placed between `octet` and `short` here in the Types list. Later in the ECMAScript type mapping section it’s placed between `unrestricted double` and `DOMString`. These lists currently use the same ordering and I think that should probably be preserved since it makes things a bit easier to follow. The latter positioning makes sense. Slotting it between two of the sized integer types seems strange:

- ± 8
- \+ 8
- ± ∞
- ± 16
- \+ 16
- ± 32
- \+ 32
- ± 64
- \+ 64